Add note for Microsoft Graph multi-tenant not supported (#6505)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/ms-graph-module.rst

| **Allowed values** | global, gcc-high, dod  |
+--------------------+------------------------+

> .. note:: At the moment, multi-tenant is not supported, so only one block of ``api_auth`` can be used in the entire configuration.

Reviewer: ```suggestion
.. note:: Multi-tenant is not supported. You can only configure one block of ``api_auth``.
```
